content.js

```diff
@@ -211,20 +206,24 @@ function resetValues() {
     //reset sponsor times
     sponsorTimes = null;
     UUIDs = null;
-    sponsorVideoID = id;
     sponsorLookupRetries = 0;
 
     //empty the preview bar
-    previewBar.set([], [], 0);
+    previewBar = null;
```

Comment:
Seems like a waste to keep creating new ones.

Reply:
So is a previewBar on a page without a previewBar xD

Reply:
On the YouTube page, the video HTML object is still always there but just hidden.

Reply:
document.querySelector("video") returned null tested in https://www.youtube.com/account

Reply:
Yes, only if you start first on a video. But maybe not for pages like that. I have only tested logged out.

Reply:
There are cases where this does not work

Reply:
Shouldn't the preview bar still be reset. How is setting it to null going to clear it?

Reply:
What do you mean? it gets removed

Reply:
Setting it to null doesn't remove it from the page I don't think

Reply:
It clears the object what did you want to happen